ftw/contacts/profiles/default/propertiestool.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<object name="portal_properties" meta_type="Plone Properties Tool">
+
+ <object name="navtree_properties" meta_type="Plone Property Sheet">
+  <property name="metaTypesNotToList" type="lines" purge="False">
+   <element value="ftw.contacts.Contact"/>
+  </property>
```

Comment:
Please also add at least the block here.
Also consider to configure `types_not_searched`.
See https://github.com/4teamwork/ftw.simplelayout/blob/master/ftw/simplelayout/profiles/default/propertiestool.xml